alembic/env.py

```diff
@@ -19,24 +19,18 @@
 # for 'autogenerate' support
 # from myapp import mymodel
 # target_metadata = mymodel.Base.metadata
-<<<<<<< HEAD
-import os
-import sys
-sys.path.append(os.getcwd())
-=======
 # target_metadata = None
 
 sys.path.append(os.getcwd())
 config.set_main_option('sqlalchemy.url', os.getenv('DEV_DATABASE_URL'))
 
->>>>>>> 886bd3f4174947f2fb32aba10b8f635d69cbd92a
 from helpers.database import Base
 from api.location.models import Location
 from api.block.models import Block
 from api.floor.models import Floor
 from api.room.models import Room
 from api.room_resource.models import Resource
-
+from api.office.model import Office
```

Comment:
I think office is the same as block, why introduce another model **office**

Reply:
I don't think they are the same thing. An office "EPIC Tower" for example could have block A, B, C etc.

Reply:
Then the relationship should also change, location is related to office, **blocks = relationship('Block')** in **location model** changes to **offices = relationship('Office') ** and office is related to blocks, **location = relationship('Location')** changes to **blocks = relationship('Blocks')** in **office model**

Reply:
I guess it should. But will I be the one to do that?

Reply:
Exactly yes. You are.

Reply:
🤨